library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -331,7 +331,7 @@ def create(snapshot_type, description, previous: nil, cleanup: nil, important: f
         out = Yast::SCR.Execute(Yast::Path.new(".target.bash_output"), cmd)
 
         if out["exit"] == 0
-          find(out["stdout"].to_i) # The CREATE_SNAPSHOT_CMD returns the number of the new snapshot.
+          all.last
```

Comment:
where is that all defined? I do not see it here

Reply:
It is another class method (just like `find`): https://github.com/yast/yast-yast2/blob/b4d77b1629166dee60515414d7f023638d16c813/library/system/src/lib/yast2/fs_snapshot.rb#L260.

Reply:
The problem is that the create snapshot command does not return the ID anymore like the installation-helper used to do.

Reply:
OK, I just a bit worry that if someone adds caching to that method it start failing, but it is NP.